perfkitbenchmarker/linux_benchmarks/cluster_boot_benchmark.py

```diff
@@ -58,7 +58,7 @@ def _GetTimeToBoot(vms, vm_index):
   """
   vm = vms[vm_index]
   metadata = {'num_cpus': vm.num_cpus, 'machine_instance': vm_index,
-              'num_vms': len(vms)}
+              'num_vms': len(vms), 'os_type': vm.OS_TYPE}
```

Comment:
Perhaps all benchmarks should include os_type.  How about completing this change and then looking to generalize.